cnvlib/call.py

```diff
@@ -170,18 +164,9 @@ def absolute_pure(cnarr, ploidy, is_reference_male):
 
 def absolute_dataframe(cnarr, ploidy, purity, is_reference_male, is_sample_female):
     """Absolute, expected and reference copy number in a DataFrame."""
-    absolutes = np.zeros(len(cnarr), dtype=np.float_)
-    reference_copies = expect_copies = np.zeros(len(cnarr), dtype=np.int_)
```

Comment:
This is a bug here (at least, I don't think this behavior was intentional). Consider this:

![absolute_dataframe_bug](https://user-images.githubusercontent.com/58295931/220121838-6b12b021-e621-422a-905d-ca893f8645e1.png)

(I.e. any change of either `reference_copies` or `expect_copies` will get reflected in both.)